README.md

```diff
@@ -121,40 +119,6 @@ git pull origin master
 python setup.py build && pip install --upgrade . && rm -rf build/
```

Comment:
Since you're updating these page, this could be changed to
```python
pip install .
```
because it now works with `pyproject.toml`.